### ISSUE TITLE
Possible Bug: Announce component - NVDA in Firefox reads such text twice or more

### ISSUE THREAD

**Issue by orion-cengage** [cengage/react-magma#602]
#### Possible bug
Need to investigate and determine whether this is a bug or not.  We received the following from an engineer:

> Hi guys! There is a problem when I'm trying to use Announce component with the text with two dynamic values. NVDA in Firefox reads such text twice or more times, as you can see on the screenshot.
> I think that the problem with NVDA. NVDA+Chrome and JAWS+Firefox work as expected.
> Any thoughts about it?
> 
> BTW, if I remove aria-atomic="true" I'll see only some separate symbols, not the whole line repeated.

![Screenshot_61](https://user-images.githubusercontent.com/24397282/156373362-a934fb2b-775b-4adf-87c0-742bc89d1e40.png)

![Nvda with Firefox](https://user-images.githubusercontent.com/24397282/156373394-bb9dee73-e97f-4e17-8884-7c2f81635578.png)

**Comment by silvalaura** [comment]
I can't replicate this issue in NVDA with Firefox. I have a feeling this is an issue with implementation; perhaps this is getting read multiple times because of the way the component is re-rendering multiple times thus triggering it repeatedly. 

Do we know who reported this so we can follow up?

**Comment by silvalaura** [comment]
This issue was reported by someone no longer active on Slack. Closing for now, and we can reopen in the future if it becomes an issue again.
